### ISSUE TITLE
jx upgrade client stuck at 2.0.219

### ISSUE THREAD

**Issue by tdcox** [jenkins-x/jx#4273]
### Summary 
```
➜ jx upgrade client
You are already on the latest version of jx 2.0.219
```
Now try brew:
```
➜ brew upgrade jx
Updating Homebrew...
--8<--
==> Upgrading 1 outdated package:
jenkins-x/jx/jx 2.0.219 -> 2.0.274
==> Upgrading jenkins-x/jx/jx
==> Downloading https://github.com/jenkins-x/jx/releases/download/v2.0.274/jx-darwin-amd64.tar.gz
```

Not sure if this was a problem with 219 or if it is an ongoing issue.

**Comment by tdcox** [comment]
Hmmm. I'm getting some odd behaviour from `jx upgrade platform` too:
```
➜ jx upgrade platform
Using provider 'gke' from team settings
? A local Jenkins X versions repository already exists, pulling the latest: Yes
using stable version 2.0.330 from charts of jenkins-x/jenkins-x-platform from /Users/terry/.jx/jenkins-x-versions
Upgrading platform from version 4 to version 2.0.330
Read credentials for https://storage.googleapis.com/chartmuseum.jenkins-x.io from vault helm/repos
WARNING: Waiting for the fork of bootsbot/environment-d38-dev to appear...
Forked Git repository to https://github.com/bootsbot/environment-d38-dev
Fetched chart jenkins-x-platform to dir /var/folders/2n/n92yy0vd2pvfj1r8r3_ntp700000gn/T/jx-helm-fetch-jenkins-x-platform-922481433/jenkins-x-platform
WARNING: No changes made to the source code in /Users/terry/.jx/environments/Bootstrap-Jersey/environment-d38-dev. Code must be up to date!
```

Latest release of jenkins-x-platform in the repo seems to be 2.0.634

**Comment by garethjevans** [comment]
This should be resolved in 2.0.277
